src/components/GlobalErrorBoundary.js

```diff
@@ -0,0 +1,72 @@
+import React, { useMemo } from 'react';
+import newGithubIssueUrl from 'new-github-issue-url';
+import { OutButton, RestartButton } from './Buttons';
+
+const buildTitle = error => `Error Report: ${error.message}`;
+
+const buildBody = error => `\n
+<!-- Thanks for submitting this error! Can you help us identify the issue by providing additional context? -->
+\n
+## Context
+\n
+<!-- 1) What were you doing when you encountered this error? -->
+\n
+<!-- 2) Only if relevant (don't self-dox!), what point were you interacting with? -->
+\n
+## Error\n\n
+\`\`\`
+${error.stack}
+\`\`\`
+\n
+`;
+
+/**
+ * NOTE: we want to use as few normal components here as possible, since the
+ * error might have originated from within those components, making this
+ * whole error boundary useless.
+ */
+export default function GlobalErrorBoundary({ error }) {
+  const url = useMemo(
+    () =>
+      newGithubIssueUrl({
+        user: 'urbit',
+        repo: 'bridge',
+        labels: ['bug', 'error handling'],
+        title: buildTitle(error),
+        body: buildBody(error),
+      }),
+    [error]
+  );
+
+  return (
+    <div className="mw1 ph4">
+      <h2 className="mt8">
+        Bridge Error!{' '}
+        <span role="img" aria-label="a bridge on fire">
+          🌉🔥
+        </span>
+      </h2>
+      <p className="mt4 f5">
+        We caught an error thrown by the Bridge application client-side code.
+        <br />
+        <b>Your points and assets are most likely safe.</b>
```

Comment:
>most likely

I'd either display this phrase with absolute confidence, or not at all.

Proposed alternative:  
"This does not affect your points and assets."

Reply:
sgtm. will wait on double-confirm from @alexmatz

Reply:
@alexmatz @vvisigoth I think this is good to go in after we get copy comments here.